README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
 <a href="https://bootstrap-vue.js.org">
-    <img src="https://github.com/bootstrap-vue/bootstrap-vue/raw/master/banner.png" width="300px">
+    <img src="https://github.com/bootstrap-vue/bootstrap-vue/raw/master/static/banner.png" width="300px">
```

Comment:
Just thinking... since this is the `dev` branch, the image will be looking in the master branch initially until merged into `master`

Reply:
Maybe for now we should leave it in root dir... or we could move it in master as well so that the url works in both dev and master.

Reply:
We can merge this right before the RC12 release to prevent conflicts.

Reply:
Will cherry-pick to master too